app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.LoginForm;
+import com.codesoom.assignment.errors.LoginDataNotMatchedException;
+import com.codesoom.assignment.utils.JwtUtil;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+/**
+ * 인증 token 정보를 관리한다.
+ */
+@Service
+@RequiredArgsConstructor
+public class AuthenticationService {
+    private final UserRepository userRepository;
+    private final JwtUtil jwtUtil;
+
+    /**
+     * 로그인 정보를 조회해서 토큰을 발급한다.
+     * @param form 이메일과 비밀번호가 포함된 로그인 정보
+     * @return 토큰
+     */
```

Comment:
javadoc에서 가장 중요한 건 main description 과 `@return`이고, 그 다음으로 중요한 것은 `@throws` 입니다.
```suggestion
    /**
     * 로그인 정보를 조회해서 토큰을 발급한다.
     * @param form 이메일과 비밀번호가 포함된 로그인 정보
     * @return 토큰
     * @throws LoginDataNotMatchedException 설명
     */
```

Reply:
- public 메서드에는 주석을 무조건 달자 
- `main description`, `@return`, `@throws`는 중요한 정보로 주석에 포함시키도록 하자.

유념하겠습니다.